app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -234,6 +235,15 @@ class HomeActivityTest {
 
   @Test
   fun testHomeActivity_recentlyPlayedStoriesTextIsDisplayed() {
+    val profileId1 = createProfileId(internalProfileId1)
```

Comment:
We can keep this in before only, and just use the profileId1 variable.

Reply:
For every tests I need to create new. profile. As it is saving progress for that profile and might be because of that testcases fails at some cases.

Reply:
ah, okay.